willowtest/specs/earthstar/es6_spec.ts

```diff
@@ -0,0 +1,293 @@
+import { Expression } from "macro";
+import { link, path, pinformative, quotes, site_template } from "../../main.ts";
+import { code, em, hr, pre } from "../../../h.ts";
+import { hsection, table_of_contents } from "../../../hsection.ts";
+import { r, rs, R } from "../../../defref.ts";
+import { marginale, sidenote } from "../../../marginalia.ts";
+import { link_name } from "../../../linkname.ts";
+import { def } from "../../../defref.ts";
+import { def_parameter_value, def_value, lis, def_fn } from "../../main.ts";
+import { field_access, function_call } from "../../../pseudocode.ts";
+import { encode_two_bit_int } from "../encodings.ts";
+
+export const es6_spec: Expression = site_template(
+		{
+				title: "Earthstar Specification",
+				name: "es6_spec",
+		},
+		[
+			table_of_contents(7),
+
+			pinformative(
+				"This document specifies version 6 of the ", link("Earthstar protocol", "https://earthstar-project.org/"), ". The protocol behind Earthstar is an instantiation of ", link_name("willow", "Willow"), ": the Earthstar data model is a particular instantiation of the ", link_name("data_model", "Willow data model"), ", using an instantiation of ", link_name("meadowcap", "Meadowcap"), " for access control, and synchronising data with the ", link_name("sync", "WGPS"), "."
+			),
+
+			pinformative(
+				"In addition to fixing a specific set of protocol parameters for Willow, Earthstar restricts the set of bytes that may appear in ", rs("Path"), " to the ascii encodings of the following characters: ", code("-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz_"), ", that is, alphanumerics and ", code("-"), ", ", code("."), ", and ", code("_"), "."
```

Comment:
I think this has been accidentally left in, unless you've had a change of heart?

Reply:
Good catch, this should go.